src/components/EntityPage/Participant/ParticipantClinical.js

```diff
@@ -55,63 +56,86 @@ class ParticipantClinical extends React.Component {
 
         diagnoses = diagnoses.map(diag => {
           //make age more readable while we wait for the calls
-          const age = diag.age_at_event_days;
-
-          const years = Number(('' + age / 365).split('.')[0]);
-          const days = age - years * 365;
-
-          function format() {
-            function y() {
-              if(years === 1) return `${years} year `;
-              else if(years === 0) return "";
-              else return `${years} years `;
-            }
-
-            function d() {
-              if(days === 1) return `${days} day`;
-              else if(days === 0) return "";
-              else return `${days} days`;
-            }
-
-            if(age === null) return "--";
-            else return y() + d();
-          }
 
-          diag.age_at_event_days = format();
+          diag.age_at_event_days = prettifyAge(diag.age_at_event_days);
 
           return diag;
         });
 
         return temp;
       })(),
     ).then(nums => {
-      for (let i = 0; i < nums.length; i++)
-        diagnoses[i].shared_with = get(nums[i], 'data.participant.hits.total', '--');
+      for (let i = 0; i < nums.length; i++) diagnoses[i].shared_with = get(nums[i], 'data.participant.hits.total', '--');
 
-      this.setState({ diagnoses: sanitize(diagnoses) });  //once we're ready, just tell the state, it'll do the rest
+      this.updateState({ diagnoses: sanitize(diagnoses) });  //once we're ready, just tell the state, it'll do the rest
```

Comment:
because it'd overwrite the phenotypes :) updateState is syntactic sugar for 

`this.setState({ diagnoses: sanitize(diagnoses), phenotypes: this.state.phenotypes})`